src/main/external-resources/renderers/Miracast-M806.conf

```diff
@@ -13,7 +13,7 @@ RendererIcon = miracast.png
 # {friendlyName=LOLLIPOP-1C3A18-DMR, manufacturer=Rockchip, modelName=Rockchip Media Renderer, modelNumber=1.0, modelDescription=Rockchip Media Renderer, DLNA(DMR), manufacturerURL=http://www.rock-chips.com, modelURL=http://www.rock-chips.com}
 # ============================================================================
 
-#MediaInfo = true
+MediaInfo = false
```

Comment:
This was commented out so I set it to false, I can change that if we think true would be good in this situation.